web/src/components/KebabMenu.tsx

```diff
@@ -76,9 +76,13 @@ const ToggleButton = styled.button`
   margin-top: 6px;
 `
 
-const KebabMenu = ({ items, direction }: KebabMenuProps): ReactElement => {
+const KebabMenu = ({ items, direction }: KebabMenuProps): ReactElement | null => {
   const [checked, setChecked] = useState<boolean>(false)
 
+  if (items.length === 0) {
+    return null
+  }
```

Comment:
Why is this change necessary?

Reply:
for the `General Header` we do not pass items, so the `KebabMenu` shouldn't be displayed. But maybe we have to remove that soon, because the footer gonna take place in the SidebarMenu.